### DISCUSSION TITLE
Increase military possibility and add a multiplayer

### DISCUSSION THREAD

**Discussion by supernewbiepro** [Keriew/augustus#823] [Ideas]
Hello. Congratulations to everyone work on this mod.
I would like suggest you idea for the military and multiplayer ;
- Add a Multiplayer mod ; each player on a PC and developp one city on the europ map like SOLO, but possibility to send troops to help the other player if he's attacked, possibility to request troop from the other player, trade with it
- For SOLO or multiplayer, possibility to attack a no roman city when we want (select on the europ map the city we would like attack and the choice could appear "attack" and "choose your troop you want send"
- Possibility to play the battle on a foreign city like when we are attacked by barbarian in our city, but we could play our roman army and begin in a corner of the city we want attack like barbarians.
- Possibility to switch between our city and city captured to developp them... And possibility to have an option "give this city to the emperor" and we lost the control of the new city captured.
- Possibility to have more soldier in each legion, and/or possibility to have more legions.
- Add a new race of soldiers, archers

It could be a very very big upgrade to do that... For information if you want, There is a old game SETTLERS 25, a german team they developed a revival of this old game, like you did with caesar 3 and AUGUTUS mod and worked wonders. Her name ; siedler 2. They added HD, they added the possibility of playing with several people, each on their PC, as allies or enemies, they added the possibility of trading between us, we couldn't do it in the original game, we can send soldiers to our allies, we couldn't do that in the original game either. They turned this old game into a nugget. You are one step away from winning the same bet. You have all my encouragement if you manage to add multiplayer and develop the military side and the conquest which has a very big potential.

**Comment by Italicus66** [comment]
Yay! Exactly this would have been one of my next questions to the developer team: Are you thinking about any kind of multiplayer option? I would like it, too! :-)

Out of topic:
@supernewbiepro
SETTLERS 25 (also known as "Return to the Roots") are here on github, too... and they appearently still active... :-D
https://github.com/Return-To-The-Roots/s25client/wiki
_Because of the independency from the Settlers-II original software a.o.  I prefer Widelands..._

**Comment by MSTVD** [comment]
Hi! On behalf of the team, we appreciate feedback and the enjoyment of Augustus from players.
That being said, while we definitely have considerations for expanding on the military aspect of the game down the line, including some of the suggestions here, we are in no way planning anything regarding multiplayer for reasons mentioned several times before in GamerZakh's discord.

**Reply by Italicus66** [reply]
Hello MSTVD,
thank you for the infos. Of course, the enjoyment about Augustus is our focus..! :-)
It may simple be that not everyone has been following the discussions from the start, or one is not on Discord; But because of your hint I'm now on Discord, too, because I wanted to know more about that multiplayer thing.

I started a search there with the keyword "multiplayer" and found a lot of statements, that would like the idea of a multiplayer, too. But I found only one specific discussion about only one specific mulitplayer idea for the last two years, can it be?
I found that discussion in the section "julius-and-augustus" from 15 August 2021started at 17:21. You were involved, too :-)
_(for those, who are not on Discord I will attache a txt-file)_

Very, very interesting, I find. It shows what problems one would get for that special idea of the member "TheWolf". At the end the moderator "Gummiel" did just not agree, and said that idea of "TheWolf", would never work...

But there could be other Multiplayer-Options. Just an easy idea like in other games, too:
-A host starts a multiplayer game and invite some other players
-Those ones appear as human players on the empire-map
-Now all human players defend the empire against the babarians as usual in sologames _(of course one should be able to help and send goods and troops to the attacking hotspots of the other players)_
-for getting a fluid gameplay all interruptions through internal city management dialogs or using the "P"-key should not be possible

Ready.. Have I forgotten something?  :-DD

_The idea is just to take social interactions inside the game, so one can laugh together about victories and defeats..._

_[Discussion about Multiuplayer in Augustus.txt](https://github.com/Keriew/augustus/files/10801964/Discussion.about.Multiuplayer.in.Augustus.txt)_

**Reply by crudelios** [reply]
We are not planning on adding multiplayer any time soon. It's a technically difficult and time-consuming effort that will divert work away from the many changes we still want to do, all in the name of what we believe to be simply an interesting gimmick.

At its core, C3 is a single-player game and we believe Augustus should share that concept.

There are, however, two ways multiplayer could be added:

- We run out of new features to add. This is highly unlikely, given the huge backlog and the relatively slow pace of feature introduction to the game;
- A new coder comes along that decides to add it himself. This is also not very likely, but still, there's hope!

Sorry if I sound harsh, I'm not actually. I just want to present our point of view as factually as possible!

**Reply by Italicus66** [reply]
Dear Crudelios,
thanks a lot for your infos and insights about that topic. No my dear, you sound not harsch, you told simply the current state, what I wanted to know. I think the most players _(and me, too!)_ are very happy about Augustus and also your new ideas exiting me...:-)) All we are talk about are only ideas and thoughts to make Rome greater one day maybe.
For me remains, the door is open and this give me a good feeling..  :-)

_Allow me to give a last thought I got last night for that missing coder: Maybe the circle does not need to be reinvented.. ..There are already free codes at other games for example Widelands.. Maybe one could just take over some things from there...:-D
Now I'm quite  :-DD Thank you a lot again..!_

**Reply by crudelios** [reply]
We'd more than welcome new willing coders!

But we can't really force anyone to come along and stay coding, and we don't have any incentive for anyone to join besides having some coding fun and the eventual community gratitude for the new features. So, it's difficult to get new coders 😉
